CMSgov/design-system
# [WNMGDS 1144] Adding storybook for local development

## Summary
Initial add of storybook for core DS

### Added
- Storybook for core DS local testing
- And example story using the Button component

## How to test
Run `yarn install` at the project root to get all the new dependencies
Run `yarn storybook` to start the storybook instance. This should open `localhost:6006` where you will see 

<img width="1792" alt="Screen Shot 2021-11-09 at 3 35 43 PM" src="https://user-images.githubusercontent.com/33579665/141016806-1a96d7de-f5b9-45d3-8a31-5df2a5e24e86.png">

There are a few Button stories to see some variations. The prop table is generated automatically, but I did some configuration to make the "controls" more specific

### Future PR Work ###
- Setting up child DS's in storybook. Talking to Patrick, we will want each DS to be separate in storybook. This will help to keep configuration & assets separate
- Setting up other add-ons
- Other component stories
